make_dataset.py

```diff
@@ -483,25 +500,60 @@ def parse_args(args_input=None):
             warnings.warn("Unable to degrade chosen excerpt from "
                           f"{comp.csv_path} and no clean excerpts requested."
                           " Skipping.", UserWarning)
-
+    
     meta_file.close()
+    
+    fh_dict = {
+        split: open(
+            os.path.join(ARGS.output_dir, f'{split}_cmd_corpus.csv'
+        ), 'w') for split in ['train', 'valid', 'test']
+    }
+    line_counts = {
+        split: 0 for split in ['train', 'valid', 'test']
+    }
+    meta_df = pd.read_csv(os.path.join('acme', 'metadata.csv'))
+    for idx, row in tqdm(meta_df.iterrows(), total=meta_df.shape[0],
+                         desc='Creating command corpus'):
+        alt_df = pd.read_csv(os.path.join('acme', row.altered_csv_path),
+                             header=None,
+                             names=['onset', 'track', 'pitch', 'dur'])
+        alt_cmd_str = df_to_command_str(alt_df)
+        clean_df = pd.read_csv(os.path.join('acme', row.clean_csv_path),
+                               header=None,
+                               names=['onset', 'track', 'pitch', 'dur'])
+        clean_cmd_str = df_to_command_str(clean_df)
+        deg_num = row.degradation_id
+        split = row.split
+        fh = fh_dict[split]
+        fh.write(f'{alt_cmd_str},{clean_cmd_str},{deg_num}\n')
+        meta_df.loc[idx, 'corpus_path'] = fh.name
+        meta_df.loc[idx, 'corpus_line_nr'] = line_counts[split]
+        line_counts[split] += 1
+    meta_df.to_csv(os.path.join(ARGS.output_dir, 'metadata.csv'))
```

Comment:
I think this should be optional with a --command flag, since it more than doubles runtime, and not everyone will want to use the command-based one-hots that we use.

We should make a function somewhere, like meta_csv_to_cmd_csvs() and just call that from the Dataset if it hasn't yet been run (or call that func from here if --command is True)

Reply:
We should also move this into the degradation loop to avoid the writing/reading the csvs multiple times, which probably causes a lot of the slowdown.

Reply:
I added --command arg and moved the func elsewhere to allow post-processing (and it works)

Reply:
Sure, though I slightly preference the default doing it anyway but we can tussle over that later.

Reply:
I actually have it as the default now (on models branch)